dvc/tree/azure.py

```diff
@@ -92,6 +93,10 @@ def blob_service(self):
             container_client.get_container_properties()
         except ResourceNotFoundError:
             container_client.create_container()
+        except HttpResponseError as e:
```

Comment:
```suggestion
        except HttpResponseError as exc:
```

Reply:
Looks like this one was forgotten :) Can't add it myself, as your organization forbids edits from maintainers in your PRs.